scripts/rebase-fork.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ -z $1 ]]; then
+    echo "Please specify the pull request number you want to rebase to a local branch."
+    echo "Usage: ./scripts/rebase-fork.sh <pr number>"
+    echo ""
+    echo "Example: ./scripts/rebase-fork.sh 557"
+    exit 1
+fi
+
+PR_NUMBER=$1
+
+declare -a req_tools=("gh" "git" "jq")
```

Comment:
I don't have `gh` installed, and when I ran the script with an invalid PR (like `abc`), I got the error `command not found: gh`